src/compat/eme/custom_media_keys/index.ts

```diff
@@ -179,6 +176,15 @@ if (isNode ||
 
     return observableThrow(undefined);
   };
+} else {
+  requestMediaKeySystemAccess = (
+    _a : string,
+    _b : MediaKeySystemConfiguration[]
+  ) : Observable<MediaKeySystemAccess> => {
+    const message = "This browser seems to be unable to play encrypted contents " +
+                    "currently (are you using HTTPS?)";
```

Comment:
Isn't it strange to ask user if he is using HTTPS ? I would rather give an advice :
`Try using HTTPS if you are not.`
Or `Playing encrypted content may not work if you're not using HTTPS`

Reply:
Done